release-team/role-handbooks/release-lead.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview:
 
-The release team leader role is responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring a retrospective happens.  The lead is also responsible for ensuring a successor is selected once the release is complete.  
+The release team leader role is responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring a retrospective happens.  The lead is also responsible for ensuring a successor is trained for future release cycles.
```

Comment:
The selection process is explicit here and not mutually exclusive to training someone else. The intent is for there to be someone committed to the next cycle, and the current release lead is responsible for ensuring that. I would say selected and trained.